src/RunProcessAsTask.Tests/packages.config

```diff
@@ -1,12 +1,12 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="coveralls.net" version="0.7.0" targetFramework="net45" developmentDependency="true" />
-  <package id="OpenCover" version="4.6.519" targetFramework="net45" />
-  <package id="xunit" version="2.1.0" targetFramework="net45" />
-  <package id="xunit.abstractions" version="2.0.0" targetFramework="net45" />
-  <package id="xunit.assert" version="2.1.0" targetFramework="net45" />
-  <package id="xunit.core" version="2.1.0" targetFramework="net45" />
-  <package id="xunit.extensibility.core" version="2.1.0" targetFramework="net45" />
-  <package id="xunit.extensibility.execution" version="2.1.0" targetFramework="net45" />
-  <package id="xunit.runner.console" version="2.1.0" targetFramework="net45" />
+  <package id="coveralls.net" version="0.7.0" targetFramework="netcoreapp1.1" developmentDependency="true" />
+  <package id="OpenCover" version="4.6.519" targetFramework="netcoreapp1.1" />
+  <package id="xunit" version="2.1.0" targetFramework="netcoreapp1.1" />
+  <package id="xunit.abstractions" version="2.0.0" targetFramework="netcoreapp1.1" />
+  <package id="xunit.assert" version="2.1.0" targetFramework="netcoreapp1.1" />
+  <package id="xunit.core" version="2.1.0" targetFramework="netcoreapp1.1" />
+  <package id="xunit.extensibility.core" version="2.1.0" targetFramework="netcoreapp1.1" />
+  <package id="xunit.extensibility.execution" version="2.1.0" targetFramework="netcoreapp1.1" />
+  <package id="xunit.runner.console" version="2.1.0" targetFramework="netcoreapp1.1" />
 </packages>
```

Comment:
Normally, packages.config becomes redundant once you switch to the new infrastructure.  Is this for travis or some other tooling?  It might be redundant.